packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -27,8 +27,10 @@ const DEFAULT_OPTIONS = {
 
   // Avatar
   avatarSize: 40,
+  botAvatarBackgroundColor: null, // defaults to accent color in styleSet
```

Comment:
Prefer to put default here, just like `suggestedActionBorder` or `suggestedActionTextColor`. I can see your points about defaulting in the style set.

TBH, we should think thoroughly on how to tackle it better, but not now:

- The dev can easily "steal" the style options JSON without looking into any logic (like `||` in your case), it feel more "definitive" and less bug prone to them ("I changed it, why it didn't change the color?")
- Although there are more places need to be changed, these types of devs are used to "replace all" (CSS modders are like this)
- Think thoroughly because we always have some "arch options" like accent color, text color, or a few things that actually seed the style options, but try not to introduce another layer

Reply:
My first thought was to set the background colors to `DEFAULT_ACCENT` in the default style options, but then the avatar background color wouldn't change when the developer changed the accent color. Would be a breaking change.

Reply:
Is this convo resolved and PR ready for merge?

Reply:
Yea I added a comment in the PR description about our offline discussion.